<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayMenu</class>
 <widget class="QWidget" name="PlayMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>160</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="hostButton">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="joinButton">
      <property name="text">
       <string>Join</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
